src/sentry/static/sentry/app/components/group/suggestedOwnerHovercard.jsx

```diff
@@ -179,6 +215,22 @@ const OwnershipTag = styled(({tagType, ...props}) => <div {...props}>{tagType}</
   text-align: center;
 `;
 
+const ViewMoreButton = styled(({onClick, ...props}) => (
+  <div {...props}>
+    <ViewMoreLink onClick={onClick} />
+  </div>
+))`
+  padding: ${space(0.25)} ${space(0.5)};
+  margin: ${space(0.25)} ${space(0.5)} ${space(0.25)} 0;
+  width: 100%;
+  text-align: right;
+`;
+const ViewMoreLink = styled(p => <a {...p}>View more</a>)`
+  color: #9586a5;
```

Comment:
Hmm should we be using a custom color?

Reply:
It's the h6 color we are using for the titles, which was the case in design, it can be gray2, they are very close